instrumentation/opentelemetry-instrumentation-tornado/src/opentelemetry/instrumentation/tornado/client.py

```diff
@@ -65,15 +65,21 @@ def fetch_async(tracer, func, _, args, kwargs):
             span.set_attribute(key, value)
 
     with trace.use_span(span):
+        if request_hook:
```

Comment:
Any difference between calling `request_hook` within the `use_span` vs outside?

Reply:
Should be called right after the span is created right?

Reply:
No difference really. I'll move it out to avoid any future confusion.